lib/Backends/OpenCL/OpenCL.cpp

```diff
@@ -124,6 +124,10 @@ OpenCLFunction::OpenCLFunction(std::unique_ptr<IRFunction> F,
   setTraceInfo(std::move(traceInfo));
 }
 
+void OpenCLFunction::freeCompilationResources() {
+  runtimeBundle_.freeConstants();
```

Comment:
Can I ask why only constants need to be freed?

Reply:
They are copied for initialization of the device after which they are no longer needed so they are freed. The symbolTable is still needed.